---
title: Software
layout: default
---
<br>

<div class="Nheader">canny</div>
<img src="assets/images/test.jpg" width="200px" class="imgright">
<p>
This software was used to analyze 2,135 Agilent 1M CGH arrays designed to assess copy number variation genotypes in Phase 3 samples from the 1000 Genomes Project.
</p>
<br>
<P> Scripts and documentation may be found at: <a href="https://github.com/mills-lab/canny/">https://github.com/mills-lab/canny/</a>.</p>
<br>
<p>Reference:<br>
<b>[Update Needed[</b>
<br>[Authors] <b><em>Journal</em></b> [Reference]</p>
<br>
<div class="Nheader">dinumt</div>
<img src="assets/images/test.jpg" width="200px" class="imgright">
<p>
This software is designed to identify and genotype nuclear insertions of mitochondrial origin from whole genome sequence data.
It consists of two programs: diumt (di-nu-mite), which identifies sites of insertions in a single sample and gnomit (geno-mite), which genotypes those sites across multiple samples.
There is an additional program name clusterNumtsVcf which will merge sites identified in multiple samples into a single merged file for genotyping.
<br>
<P> Source code and documentation is available at <a href="https://github.com/mills-lab/dinumt/">https://github.com/mills-lab/dinumt/</a>.</p>
<br>
<p>Reference:<br>
<b>[Update Needed]</b>
<br>[Authors] <b><em>Journal</em></b> [Reference]</p>
<br>
<div class="Nheader">F-seq</div>
<img src="assets/images/test.jpg" width="200px" class="imgright">
<p>
This software is designed to identify both simple and complex rearrangements from paired-end sequencing data.
</p>
<br>
<P>Source code and detailed documentation is available at <a href="https://github.com/mills-lab/svelter/">https://github.com/mills-lab/svelter/</a>.</p>
<br>
<p>Reference:<br>
<b>[Update Needed]</b>
<br>[Authors] <b><em>Journal</em></b> [Reference]
</p>
<br>
<div class="Nheader">SPECtre</div>
<img src="assets/images/test.jpg" width="200px" class="imgright">
<p>
This software is designed to identify regions of active translation from ribsome profiling sequence data.
This analytical pipeline scores the translational status of each annotated region (5'UTR, CDS, exon, 3'UTR) as a function of its spectral coherence over user-defined N nucleotide sliding windows to an idealized reference coding signal.
It is used by calling <em>SPECtre.py</em> with the listed parameters.
<br>
<p> Source code and documentation is available at <a href="https://github.com/mills-lab/spectre/">https://github.com/mills-lab/spectre/</a>.</p>
<br>
<p>Reference:
<br>
<b>[Update Needed]</b>
<br>[Authors] <b><em>Journal</em></b> [Reference]
</p>
